src/components/TodoFooter/TodoFooter.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const TodoFooter = (props) => {
```

Comment:
`Todo` + `Footer`
А где тут Todo?

Либо 
```suggestion
export const TodoAppFooter = (props) => {
```
```suggestion
export const Footer = (props) => {
```